New task to prepare training data to tag s-jets

lorber98 opened the pull request:
I added a new task (sjetTreeCreator.cxx) in PWGLF/Tasks/Strangeness that creates TTree with information about the jets to be used as input for machine learning. The aim is to tag jets originating from strange quarks. This new task is inspired by PWGJE/Tasks/bjetTreeCreator.cxx.
I also added a method getSJetFlavor() to PWGJE/Core/JetTaggingUtilities.h that also returns the strange flavour.

alibuild commented:
Error while checking build/O2Physics/o2 for e6def02d99b45ebf4b3a44e2d8606bcd03e7b349 at 2025-08-16 12:11:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/12599-slc9_x86-64/0/PWGLF/Tasks/Strangeness/sjetTreeCreator.cxx:423:46: error: unused parameter 'origParticles' [-Werror=unused-parameter]
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/12599/e6def02d99b45ebf4b3a44e2d8606bcd03e7b349/build_O2Physics_o2/pretty.html).

nzardosh commented:
@hahassan7 can youm please check that this doesnt interfere with any of your stuff? I will then approve it

nzardosh commented:
@hahassan7 I think you missed the previous message, could you check again please?

lorber98 commented:
> @hahassan7 I think you missed the previous message, could you check again please?

Dear @nzardosh, thank you very much for your availability.

In principle this pull request shoud not interfere with other tasks.
I added only two elements to the enum JetTaggingSpecies in PWGJE/Core/JetTaggingUtilities.h and I added a new method getSJetFlavor() to the same file.
In any case, let's also see what @hahassan7 says about that.

Thank you both.
All the best
